Rmarkdown/Adverse_Events.Rmd

```diff
@@ -46,6 +38,7 @@ my.data <- read.csv("../Data/AE_xgx.csv")
 my.data$DOSE_label <- paste(my.data$Dose,"mg")
```

Comment:
What are the required variable  names for this dataset? Adding a block of variable name definitions like in the other pages would help the user to understand what variables are needed.

Reply:
Include DOSE_label & AE_binary definitions in this block of definitions